### ISSUE TITLE
passing normal props with create container

### ISSUE THREAD

**Issue by gilesbradshaw** [ericclemmons/react-resolver#46]
link and linkName are just normal non async props. - is there/could there be a better way to pass them than..

`````
export var YoAlt = Resolver.createContainer(InnerComponent, {
  resolve: {
    linkName:()=>"APPPPPP",
    link:()=>'app1',  
    list: (props, context)=> 
     new Promise(
        resolve=>
          setTimeout(
            (()=>
              resolve(new List([{id:0,content:"ahhhh"}]))
              ),
            1000)
        )
  }
});

`````

for example maybe

`````
export var YoAlt = Resolver.createContainer(InnerComponent, {
  props:{
    linkName:'APPPPP',
    link:'app1'
  },
  resolve: {
     list: (props, context)=> 
     new Promise(
        resolve=>
          setTimeout(
            (()=>
              resolve(new List([{id:0,content:"ahhhh"}]))
              ),
            1000)
        )
  }
});

````

**Comment by ericclemmons** [comment]
Makes sense to me!  I can adjust this...

**Comment by iamdustan** [comment]
Is this different than #36?

**Comment by gilesbradshaw** [comment]
Looks like it is addressing the same issue @iamdustan thanks!

**Comment by ericclemmons** [comment]
Fixed in v1.2.0 & v1.2.1 (#36)
